src/pages/_app.tsx

```diff
@@ -0,0 +1,8 @@
+import '../styles/globals.css';
```

Comment:
Do we have any plan for [module-path-aliases](https://nextjs.org/docs/advanced-features/module-path-aliases)? When the project gets larger, it really helps with importing by calling `@/styles/**`!